### DISCUSSION TITLE
Stop rows from being selectable

### DISCUSSION THREAD

**Discussion by elliotSMK** [Megabit/Blazorise#2538] [Q&A] [answered]
Hi, I'm just wondering if there's a way to stop rows from being selectable. I can see that they get the `table-row-selectable` added to each row, is there an option to toggle this off?

**Comment by stsrki** [comment, marked as answer]
Yes, there is. Use RowSelectable event, eg.

```cshtml
RowSelectable="@((item)=>item != null && someOtherValue)"
```

**Reply by elliotSMK** [reply]
I see that this stops the row from doing any transition when selected. But is there a way to stop the pointer appearing on hover and the row from highlighting in purple when clicked? We'd like to have just normal `table-row` without the selectable part at all if possible

**Reply by stsrki** [reply]
Maybe try with styling

```cshtml
RowStyling="@((item, e)=>e.Style = "cursor: default;")"
```

**Reply by elliotSMK** [reply]
Yes! Thank you, the styling changes worked, it might be nice to have a bool toggle for selectable rows if it's possible in the future. In case anyone comes to this thread looking for the same thing, this was the styling that worked for me:
`SelectedRowStyling="@((item, css) => { css.Background = Background.None; css.Class = "table-row"; css.Color = Color.None; })"`
`RowHoverCursor="@((item) => Cursor.Default )"`
